### PR TITLE
Created usecases/sai-ptf/ that will be used to run SAI PTF TCs

### PR DESCRIPTION
**Pull request by andriy-kokhan**

### CODE REVIEW

File: usecases/sai-ptf/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+```
+
+4. Run a test
+
+
+## Via pyTest
+
+Run a PTF test
+```
+pytest --testbed=saivs_thrift_standalone usecase/sai-ptf/SAI/ptf/saifdb.py -k FdbAttributeTest -v
```
Comment (taras-keryk-plv) [inline usecases/sai-ptf/README.md:37]: maybe should be  - usecases

Comment (andriy-kokhan) [reply usecases/sai-ptf/README.md:37]: good catch

Comment (andriy-kokhan) [reply usecases/sai-ptf/README.md]: fixed

File: usecases/sai-ptf/ptf-conftest.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+def set_ptf_params(request):
+    if request.config.option.testbed:
+        tb_params = SaiTestbedMeta("/sai-challenger", request.config.option.testbed)
+        ports = to_ptf_int_list(tb_params.config['dataplane'][0]['port_groups'])
+    else:
+        ports = ""
+    
+    arg_back = sys.argv
+    # provide required PTF runner params to avoid exiting with an error
+    sys.argv = ['ptf.py','--test-dir', '/sai-challenger/sai/ptf', *ports]
```
Comment (taras-keryk-plv) [inline usecases/sai-ptf/ptf-conftest.py:22]: should be /sai-challenger/usecases/sai-ptf/SAI/ptf ?

Comment (andriy-kokhan) [reply usecases/sai-ptf/ptf-conftest.py:22]: yeh... thanks

**Pull request merged**